src/components/common/chips.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  View,
+  TextProps,
+  ImageProps,
+  ViewProps,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+
+interface ComponentProps {
+  children: React.ReactElement<TextProps | ImageProps> | React.ReactElement<TextProps | ImageProps>[];
```

Comment:
type ChildElement = React.ReactElement<TextProps | ImageProps>;
...
children: ChildElement | ChildElement[];

Reply:
even better to move icon elements in prop, like we do it in framework

Reply:
Done.